services/web-app/server.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+//
+// The purpose of this file is to allow https localhost development.
+// https://medium.com/responsetap-engineering/nextjs-https-for-a-local-dev-server-98bb441eabd7
+//
+const { createServer } = require('https')
+// eslint-disable-next-line node/no-deprecated-api
+const { parse } = require('url')
+const next = require('next')
+const fs = require('fs') // this is a test
+const { PRODUCTION, getRunMode } = require('@carbon-platform/api/run-mode')
+
+const PORT = 443
+
+const dev = getRunMode !== PRODUCTION
+const app = next({ dev })
```

Comment:
Note: I can't remember if it's DEV or DEVELOPMENT at this point in the repo
```suggestion
const isDevMode = getRunMode() === DEV
const app = next({ dev: isDevMode })
```